### ISSUE TITLE
Progress bar for ServiceX executor coffea

### ISSUE THREAD

**Issue by alexander-held** [ssl-hep/ServiceX#419]
# Story
As an analyzer, I want to track the progress of my analysis pipeline when using the ServiceX executor in coffea. This is different from the transform / download progress bars of ServiceX: I am looking for a single progress bar to track the overall progress of all processor instances that will need to run.

# Assumptions
1. Information regarding progress is available, including the total number of processors that need to run (that might not exist at first?).

# Acceptance Criteria
1. A progress bar appears when using coffea with the ServiceX executor, showcasing the progress of processors. It looks the same as it does when running coffea without ServiceX.

**Comment by ponyisi** [comment]
Hi @alexander-held - is this still relevant?

**Comment by alexander-held** [comment]
No, the executor is no longer a thing in coffea so this can be closed.
